spec/takeaway_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'takeaway'
+require 'dotenv'
+
+describe Takeaway do
+  subject(:takeaway) { described_class.new(menu, calculator, twilio) }
+  let(:calculator) { double :calculator }
+  let(:choice) { {beef:2, rolls:3} }
+  let(:twilio) { double :twilio }
+  let(:menu) { double :menu, show: {ribs: 3,
+    beef: 4,
+    rolls: 3,
+    chips: 2,
+    pies: 5}
+    }
```

Comment:
Lint/BlockAlignment: `}` at 14, 4 is not aligned with `let(:menu) { double :menu, show: {ribs: 3,` at 9, 2.